graphics/rawtherapee/Portfile

```diff
@@ -0,0 +1,199 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           active_variants 1.1
+PortGroup           app 1.0
+
+name                rawtherapee
+app.name            RawTherapee
+version             5.8
+categories          graphics
+platforms           darwin
+license             GPL-3
+maintainers         @jasonliu-- openmaintainer
+
+homepage            http://rawtherapee.com
```

Comment:
please use `https` here and for `master_sites`

Reply:
I didn't notice that the URLs were `http`. I just copied them directly from their GitHub page.

In any case, I made the change.